src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -427,6 +436,24 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			{ action: this._runAllCellsAction },
 			{ action: clearResultsButton }
 		]);
+	}
+
+	protected initNavSection(): void {
+		this.addButton(localize('previousButtonLabel', "Previous"),
```

Comment:
As mentioned, buttons should only appear if:
- We have a navigation provider. This is important or we'll start showing these buttons even without your Book provider available. Note: Please also guard your Book navigation implementation so it's only added if in Dev!
- The provider states it hasNavigation available for this notebook. Again, otherwise we should hide or this will appear for all notebooks.

So the checkin criteria are:
- The buttons should not show in Insiders / Stable yet (only add the navigation provider if in Dev, and here don't show unless we have one)
- The buttons shouldn't show in Dev unless a book is opened (e.g. the navigation provider states this notebook is in a book).